source/multiple-mongosyncs.txt

```diff
@@ -41,7 +41,8 @@ Configure Multiple ``mongosync`` Instances
 
 The number of ``mongosync`` instances must match the number of shards on
 the source cluster. For a replica set source, you can only use one
-``mongosync`` instance. 
+``mongosync`` instance. You must use the same version of ``mongosync``
```

Comment:
[Question]

Would it make more sense to put this sentence after "The number of ``mongosync`` instances must match the number of shards on the source cluster." ?

This seems to be where we're talking about using multiple ``mongosync`` instances, so it might make sense to have this info there rather than where we're talking about only using one ``mongosync`` instance.

Reply:
That makes sense, just made the switch!